risedev.yml

```diff
@@ -1431,20 +1431,28 @@ template:
     # address of redis
     address: "127.0.0.1"
 
-  # MySQL service. Currently user-managed only
+  # MySQL service backed by docker.
   mysql:
     # Id to be picked-up by services
-    id: mysql
+    id: mysql-${port}
```

Comment:
How is this `port` determined?

Reply:
It'll be expanded to the `port` field. May refer to other components.

https://github.com/risingwavelabs/risingwave/blob/60b7a82b6ad5f952f583c56cee989d8c009cf502/risedev.yml#L1105-L1106

Will be expanded with `DollarExpander`.

https://github.com/risingwavelabs/risingwave/blob/60b7a82b6ad5f952f583c56cee989d8c009cf502/src/risedevtool/src/config/dollar_expander.rs#L21-L25